src/openforms/forms/utils.py

```diff
@@ -138,9 +140,25 @@ def import_form(import_file, existing_form_instance=None):
     return import_form_data(import_data, existing_form_instance)
 
 
-@transaction.atomic
+@contextmanager
+def default_language(language: str, *args, **kwargs):
+    activate(language)
+    try:
+        yield
+    finally:
+        deactivate()
```

Comment:
we now have some duplication and slightly different ways of doing things, see `openforms.forms.api.viewsets.FormViewSet.retrieve` where the following code is used:

```py
if not form.translation_enabled and not is_admin_request(request):
    translation.activate(settings.LANGUAGE_CODE)
```

IMO `settings.LANGUAGE_CODE` is the correct setting to use rather than `MODELTRANSLATION_DEFAULT_LANGUAGE`, as it means this is our default language in the application. The model translation setting is used internally in their fallback logic for when a translation for the active language is missing.

Additionally, this should be a utility that abstracts away the implementation details so that i can be used in both the import and viewset, something like:

```py
def activate_default_language() -> None:
    ...

@contextmanager
def force_default_language():
    ...
```

P.S. also look into `contextlib.ContextDecorator`, so you can use the utility both as context manager and as decorator, this would allow you to do:

```py
@transaction.atomic
@force_default_language
def import_form_data(...):
    ...
```